lib/flipper/ui/action_collection.rb

```diff
@@ -1,20 +0,0 @@
-module Flipper
```

Comment:
I'd actually rather just duplicate what we need for the API gem for now and we can extract things later. Let's completely duplicate and worry about tidying up at the end. Seem ok? Any strong feelings against that?